src/app/modules/zxing-scanner/browser-code-reader.ts

```diff
@@ -137,6 +156,27 @@ export class BrowserCodeReader {
         this.videoElement.addEventListener('loadedmetadata', this.videoLoadedMetadataEventListener);
     }
 
+    private checkTorchCompatibility(): void {
```

Comment:
Should return something, a Observable, Promise, something to get the compatibility test value.